packages/cloud_functions/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* Initial release
```

Comment:
Could you add more description on what is in the initial release?

Reply:
Done